posthog/warehouse/models/data_modeling_job.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.utils import timezone
+from posthog.models.utils import CreatedMetaFields, UUIDModel, UpdatedMetaFields
+
+
+class DataModelingJob(CreatedMetaFields, UpdatedMetaFields, UUIDModel):
+    class Status(models.TextChoices):
+        RUNNING = "Running", "Running"
+        COMPLETED = "Completed", "Completed"
+        FAILED = "Failed", "Failed"
+
+    team = models.ForeignKey("posthog.Team", on_delete=models.CASCADE)
+    saved_query = models.ForeignKey("posthog.DataWarehouseSavedQuery", on_delete=models.CASCADE)
```

Comment:
We won't want to delete these rows when a `DataWarehouseSavedQuery` gets deleted - instead we'd likely wanna set the `saved_query_id` to null or something instead